cmd/node/factory/structs.go

```diff
@@ -484,12 +484,28 @@ func createAntifloodComponent(mainConfig *config.Config) (consensus.P2PAntiflood
 
 	maxMessagesPerPeer := mainConfig.Antiflood.PeerMaxMessagesPerSecond
 	maxTotalSizePerPeer := mainConfig.Antiflood.PeerMaxTotalSizePerSecond
+	maxMessages := mainConfig.Antiflood.MaxMessagesPerSecond
```

Comment:
for extra clarity, I'd keep the "PerSecond" ending (if that's the intent). otherwise, it can become hard to follow the logic of "what is maxMessagesPerPeer doing and how".

Reply:
changed them exactly as they were defined in mainConfig.Antiflood struct